components/Card.tsx

```diff
@@ -49,15 +49,15 @@ const Card = ({
     "border cursor-pointer": !w.revealed,
     "border-4": w.revealed,
     "border-gray-600 bg-gray-200 hover:bg-gray-100": !w.shown,
-    "border-red-800 bg-red-700 text-red-300":
+    "border-red-700 bg-red-500 text-red-500 hover:bg-red-700 hover:text-red-100":
```

Comment:
Thanks for your PR Michael. I agree this would make it closer to the physical version to hide clicked-on words. 

However I would suggest removing the text altogether than using a text color that matches the background? Your solution would not work with the emoji version, for instance. 

This would mean touching this line instead: 
https://github.com/benjamintd/codenames.cards/pull/27/files#diff-79295464f29862a18dbcd4f252c0850bR99
to something like this: 
```
{w.revealed && <div>...</div>}
```

Let me know if you feel like doing this in this PR?

Reply:
I went with an alternative solution that allows the text or emoji to be shown again when the card is hovered.